src/index.ts

```diff
@@ -183,7 +184,8 @@ const axios = Axios.create({
                 id: stackToUpdate.Id,
                 StackFileContent: composeFile.toString(),
                 Env: composeEnvArray,
-                Prune: true
+                Prune: true,
+                Pull: !!forcePull,
```

Comment:
From the mentioned documentation I see the name 'pullImage' instead of 'pull'. Are you sure it is correct?

Reply:
It's not correct! I forgot to push a commit. I am really sorry. Fix it now.